### PR TITLE
color changes & move to constants for library coloring

### PR DESCRIPTION
**Pull request by ahsu1230**
@andrewconner @yipingliao @stkem 

keep card attribution to libraries was always green -> now consistent with their respective visibility
and constants =)

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/common/directives/keepWho/keepWhoLib.tpl.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <span>
-  <a class="kf-keep-who-pic-box" ng-href="{{library.path}}">
+  <a class="kf-keep-who-pic-box" ng-href="{{library.path}}" ng-class="{ 'secret': library.secret == true}">
```
Comment (yipingliao) [inline kifi-backend/modules/shoebox/angular/src/common/directives/keepWho/keepWhoLib.tpl.html]: Using `==` is the same as just `library.secret`, which tests for truthy. Is truthy good enough, or did you mean `===`?

**Comment by yipingliao** [comment]
Yay!

**Comment by stkem** [comment]
Thanks!

File: kifi-backend/modules/shoebox/angular/src/common/directives/keepWho/keepWhoLib.tpl.html (hunk 1 above)
Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/common/directives/keepWho/keepWhoLib.tpl.html]: You didn't add this in this PR, but why is there a `secret` field on library anyways. Can we just check visibility?

File: kifi-backend/modules/shoebox/angular/src/common/build-css/constants.styl [hunk 2]
```diff
@@ -13,6 +13,8 @@ darkBlueColor = #478eeb
 greenColor = #41c482
 kifiBlack = #333
 kifiHeadColor = #99ccf2
+libraryRed = #f06463
+libraryGreen = #73c785
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/common/build-css/constants.styl:17]: Thanks!

File: kifi-backend/modules/shoebox/angular/src/common/directives/keepWho/keepWhoLib.tpl.html (hunk 1 above)
Comment (ahsu1230) [inline kifi-backend/modules/shoebox/angular/src/common/directives/keepWho/keepWhoLib.tpl.html]: So for some reason, there isn't a visibility field at this level. It must've gotten taken out somewhere & substituted with secret boolean field

Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/common/directives/keepWho/keepWhoLib.tpl.html]: I see.

**Pull request merged**